macroslib/Cargo.toml

```diff
@@ -23,6 +23,8 @@ rustc-hash = "1.0.1"
 smol_str = "0.1.11"
 strum = "0.15.0"
 strum_macros = "0.15.0"
+cpython = { version = "0.2", features = ["extension-module"] }
```

Comment:
Why macroslib need this? It is just generates code.

Reply:
Removed